composer.json

```diff
@@ -32,6 +32,7 @@
         "guzzlehttp/guzzle": "^7.3",
         "laravel/framework": "^7.20|^8.19",
         "orchestra/testbench-core": "^5.0|^6.0",
+        "phpstan/phpstan": "^0.12.93",
```

Comment:
We could also remove this and use a global install of phpstan, so the users don't have to download phpstan if they don't use it. We're currently not using any extensions so this should be possible.

Reply:
Thanks for your suggestion. I'm ok with it being a dependency here.